src/main/markdown/release-notes.md

```diff
@@ -98,11 +98,12 @@ The GWT Release Notes
 - Add support for JsType class in UIBinder.
 
 ### JDK emulation
-- Emulate java.io.UncheckedIOException
-- Emulate Optional<T> and its int, long, double variants
-- Emulate Objects.requireNonNull() with message Supplier
+- Emulate java.io.UncheckedIOException.
+- Emulate Optional<T> and its int, long, double variants.
```

Comment:
We should put class names into backticks; at least for `Optional<T>`, the `<T>` is interpreted as HTML otherwise (and fortunately just made invisible without consequence on the formatting of the page)